Zend/zend_operators.c

```diff
@@ -792,7 +809,18 @@ ZEND_API void ZEND_FASTCALL convert_to_object(zval *op) /* {{{ */
 }
 /* }}} */
 
-ZEND_API zend_long ZEND_FASTCALL zval_get_long_func(zval *op) /* {{{ */
+/* Shut-up GCC and Clang for our custom format specifier */
+_Pragma("GCC diagnostic push")
+_Pragma("GCC diagnostic ignored \"-Wformat\"")
+_Pragma("clang diagnostic ignored \"-Wformat-invalid-specifier\"")
+_Pragma("GCC diagnostic ignored \"-Wformat-extra-args\"")
```

Comment:
Ewww. I think you'd be better off adding zend_error_unchecked.

Reply:
Yeah... this becoming pragma hell